spmd/testing/dtensor_lagging_op_db.py

```diff
@@ -312,6 +293,25 @@
     ("lu_unpack", ""),
     ("mH", ""),
     ("mT", ""),
+    ("masked.amax", ""),
```

Comment:
what's the pointwise op that is really enabled in this PR? it looks like we just re-generate the op db with additional entries of operator tests, but nothing get removed from this `xfail` list.

Reply:
A lot of things got removed from the xfail list.

Reply:
All of these were removed from xfail. I was given the mandate of "as many as I could"; and this was me pulling on that and related strings.

And I asked you and @aazzolini a fair number of questions.

If you don't want me contributing, ok. 

```
    xfail("clamp"),
    xfail("clamp_max"),
    xfail("clamp_min"),
    xfail("fmax"),
    xfail("fmin"),
```